contracts/policies/PowerEventReplacement.sol

```diff
@@ -22,12 +22,34 @@ contract PowerEventReplacement is PowerEvent {
     nutzAddr = contr.nutzAddr();
     initialSupply = 2399896170149257466012; //initialSupply as per old Power Event
     initialReserve = 22469750000000000000;  // initialReserve as per old Power Event
-    uint256 ceiling = contr.ceiling();
```

Comment:
Removed `moveCeiling()` in replacement event because it is redundant. `contr.moveCeiling(1000000)` has no meaning.